docs/topics/api/reviewers.rst

```diff
@@ -168,7 +168,7 @@ This endpoint allows you to browse through the contents of an Add-on version.
     :>json string validation_url_json: The absolute url to the addons-linter validation report, rendered as JSON.
     :>json string validation_url: The absolute url to the addons-linter validation report, rendered as HTML.
     :>json boolean has_been_validated: ``True`` if the version has been validated through addons-linter.
-    :>json object addon: A simplified :ref:`add-on <addon-detail-object>` object that contains only a few properties: ``id``, ``name``, ``icon_url`` and ``slug``.
+    :>json object addon: A simplified :ref:`add-on <addon-detail-object>` object that contains only a few properties: ``id``, ``name``, ``icon_url`` and ``slug``. ``slug`` can be null.
```

Comment:
Maybe we could expand this `addon` object like we do for `file` below, what do you think?